package.json

```
@@ -41,13 +41,14 @@
   },
   "peerDependencies": {
     "intl": "^1.2.5",
-    "react-intl": "^2.8.0"
+    "react-intl": ">2.8.0"
```

Hm, thinking about this, should we instead specify specifically the v2 version and v5 version? We wouldn't support v6 blankly and not really a point to support v 3 or 4.
```suggestion
    "react-intl": "^2.8.0 || ^5.0.0"
```

---

at a minimum we should match the range we specified in the other packages
https://github.com/cerner/terra-framework/blob/main/packages/terra-application-navigation/package.json#L31